src/apiClient.ts

```diff
@@ -40,7 +40,7 @@ export default function APIClient(
         if (!this.uploader) {
             const millis: number = parseNumber(mpInstance._Helpers.getFeatureFlag(
                 Constants.FeatureFlags.EventBatchingIntervalMillis
-            ));
+            ) as string);
```

Comment:
Can you add this so we can fix up the feature flags later?

https://go.mparticle.com/work/SQDSDKS-6317